src/internal/hooks/use-visual-mode/__tests__/use-visual-mode.test.tsx

```diff
@@ -83,11 +74,33 @@ describe('useVisualRefresh', () => {
     return <div data-testid="current-mode">{isRefresh.toString()}</div>;
   }
 
-  test('should return false when CSS-variables are not supported', () => {
-    (window.CSS.supports as jest.Mock).mockReturnValue(false);
+  beforeEach(() => clearVisualRefreshState());
+  afterEach(() => document.body.classList.remove('awsui-visual-refresh'));
+  afterEach(() => jest.restoreAllMocks());
+
+  test('should return false when class name is not present', () => {
     render(<App />);
     expect(screen.getByTestId('current-mode')).toHaveTextContent('false');
   });
+
+  test('should return true when class name is present', () => {
+    document.body.classList.add('awsui-visual-refresh');
+    render(<App />);
+    expect(screen.getByTestId('current-mode')).toHaveTextContent('true');
+    document.body.classList.remove('awsui-visual-refresh');
```

Comment:
Removed by the `afterEach` function.

Reply:
Right. Leftover from the past iterations